docs/src/main/sphinx/object-storage/metastores.md

```diff
@@ -497,6 +500,18 @@ iceberg.catalog.type=rest
 iceberg.rest-catalog.uri=http://iceberg-with-rest:8181
 ```
 
+`iceberg.security` must be `read_only` when connecting to Databricks Unity catalog
+using an Iceberg REST catalog:
```

Comment:
maybe also include iceberg.rest-catalog.namespace in the example, even though its 
optional, with a non-default other than `main` ?

Reply:
Actually, `main` isn't a default value. Added `iceberg.rest-catalog.namespace` config property to the example anyway.